proxy-node-shared/src/main/java/uk/gov/ida/notification/validations/PEMCertificateValidator.java

```diff
@@ -12,9 +12,10 @@
 
 public class PEMCertificateValidator implements ConstraintValidator<ValidPEM, String> {
 
-    // self-signed test cert is ~900-1000 chars
+    // Self-signed test cert is ~900-1000 chars.
+    // NL cert comes in at just over 2070 chars.
```

Comment:
Tom's right - we don't put comments in the code except in very rare circumstances where it makes sense. We can get rid of these